rocketpy/Flight.py

```diff
@@ -2546,14 +2512,19 @@ def calculate_rail_button_forces(self):
         F22: Function
             Rail Button 2 force in the 2 direction
         """
+        # First check for no rail phase or rail buttons
+        nullForce = []
```

Comment:
Actually, it is important that even the nullForce to be represented as a Function object with value 0 and length higher than 1. Otherwise I'm afraid the calculations of forces (in the funcified cached properties) will not work properly.
See for example line 2295.